### PR TITLE
Fixing EVSSClaim references

### PR DESCRIPTION
**Pull request by knkski**
Allows running just the tests located in `evss_claim_service_spec.rb` with `bundle exec rspec -P spec/services/evss_claim_service_spec.rb`. I'm guessing this currently works when all tests are run because it's already imported in other tests.

### CODE REVIEW

**Comment by austin-martinez** [comment]
The file should be autoloaded, it looks like the class name here is incorrect and doesn't match the path
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/evss/document_upload.rb
It should just be `EVSS::DocumentUpload`

**Comment by knkski** [comment]
@austin-martinez: Cool, thanks for the help. Should `EVSSClaim::RequestDecision` also get changed to just `EVSS::RequestDecision`?

File: app/workers/evss/document_upload.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-class EVSSClaim::DocumentUpload
+class EVSS::DocumentUpload
```
Comment (austin-martinez) [inline app/workers/evss/document_upload.rb]: There will probably be workers stuck in the queue with this class (failed to upload and are being retried), you could either update those workers with a rake task/migration to use this new class, or create a new class with the old name and pass through all the args to the new class. The other class is less likely to have failures but probably a good idea to follow the same migration path

File: app/workers/evss/document_upload.rb [hunk 2]
```diff
@@ -12,3 +12,13 @@ def perform(auth_headers, user_uuid, document_hash)
...
   end
 end
+
+# Allows gracefully migrating tasks in queue
+# TODO(knkski): Remove after migration
+class EVSSClaim::DocumentUpload
+  include Sidekiq::Worker
+
+  def perform(auth_headers, user_uuid, document_hash)
+    EVSS::DocumentUpload.new.perform(auth_headers, user_uuid, document_hash)
```
Comment (markolson) [inline app/workers/evss/document_upload.rb]: Let's change this to a `perform_async` so it removes it from the `EVSSClaim::DocumentUpload` queue and adds it as an `EVSS::DocumentUpload` job

**Comment by markolson** [comment]
@knkski @austin-martinez can take a peek at 639b3a3 and 👍 or 👎 the re-queue mechanism? It should drain the `EVSSClaim` over time.

**Pull request merged**